x/ibc/02-client/keeper/encoding.go

```diff
@@ -1,90 +1,33 @@
 package keeper
 
 import (
-	"fmt"
-
-	"github.com/cosmos/cosmos-sdk/codec"
 	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/exported"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/simulation"
+	"github.com/cosmos/cosmos-sdk/x/ibc/02-client/types"
 )
 
+var _ simulation.ClientUnmarshaler = (*Keeper)(nil)
```

Comment:
it's weird to me at least to import simulation package from the keeper

Reply:
It's only to check that keeper implements the desired interface

Reply:
Moved this interface check to simulation